searx/static/themes/simple/src/js/main/results.js

```diff
@@ -16,6 +16,36 @@
           }
         ));
 
+    function ping (url, callback) {
+      var start = performance.now();
+      var faviconUrl = new URL('/favicon.ico', url).href;
+      var xhr = new XMLHttpRequest();
```

Comment:
I don't think we want to ping every result item .. I see a violation of the privacy aspect in SearXNG .. with this ping the IP from the user is send to every domain in the result list, long before the user has decided to follow a link from the results.